doc/release_notes.rst

```diff
@@ -23,6 +23,8 @@ This part of documentation collects descriptive release notes to capture the mai
 
 * Update workflow to geopandas >= 1.0 `PR #1276 <https://github.com/pypsa-meets-earth/pypsa-earth/pull/1276>`__
 
+* Introduce universal currency conversion
+
```

Comment:
It would be great to adjust the note formatting to match the format of the others for consistency

Reply:
Perfect, thank :)